order.go

```diff
@@ -196,6 +208,28 @@ type Transaction struct {
 	PaymentDetails *PaymentDetails  `json:"payment_details"`
 }
 
+type Fulfillments struct {
```

Comment:
Could you change this to singular `Fulfillment`?

Reply:
@dlebech Done! 😃